CHANGELOG.md

```diff
@@ -4,8 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<!-- BEGIN RELEASE NOTES -->
 ### [Unreleased]
 
+### [2.3.0] - 2023-09-29
```

Comment:
This is a patch version. 
The version number cannot be bumped to 2.3.0. 
```suggestion
### [2.2.0+v3api] - 2023-09-29
```
The above adheres to semver and adds metadata around the patch. This denotes that this is essentially a patch for a specific version with no additional functionality but support for V3 APIs.

Adding metadata is allowed according to semver, you can test using this [regex validator](https://regex101.com/r/Ly7O1x/3/). https://semver.org/

![image](https://github.com/klaviyo/magento2-klaviyo/assets/32624393/4aa2d82d-a091-4610-bb2c-4b300e779ca7)